abci/proposal_auction.go

```diff
@@ -152,15 +152,6 @@ func (h *ProposalHandler) buildTOB(ctx sdk.Context, bidTx sdk.Tx, maxBytes int64
 		return proposal, err
 	}
 
-	maxBytesForLane := utils.GetMaxTxBytesForLane(
```

Comment:
Removing redundant code the is already in the proposal type.

Reply:
👍